factory/templates/vconsole.conf.in

```diff
@@ -0,0 +1,3 @@
+# This is the fallback vconsole configuration provided by systemd.
+
+KEYMAP={{ SYSTEMD_DEFAULT_KEYMAP }}
```

Comment:
it might be ok to add this, but this should really be a commented line, so tha the file is without effect and the built-in default takes effect, not the config defaul